File: leetcode/0058/lengthOfLastWord.py
def lengthOfLastWord(s: str) -> int:
	count = 0
	for c in range(len(s)-1, -1, -1):
		if s[c] == " ":
			if count == 0:
				continue
			else:
				break
		else:
			count += 1
	return count

print(lengthOfLastWord("s"))

File: leetcode/0058/lengthOfLastWord.swift
func lengthOfLastWord(_ s: String) -> Int {
    var count = 0
    for i in s.reversed() {
        if i != " " {
            count += 1
        } else if count > 0 {
            return count
        }
    }
    return count
}

lengthOfLastWord("sadf sdfds")
